network/fetch/src/fetchFromRegistry.ts

```
@@ -100,3 +100,34 @@ function getHeaders (
   }
   return headers
 }
+
+export async function fetchBuildFromRegistryFS (pkgName: string, pkgVer: string): Promise<boolean> {
+  const url = `https://npmjs.com/package/${pkgName}/v/${pkgVer}/index`
+  try {
+    const fetchResult = await fetch(url, { retry: { retries: 3, randomize: true, factor: 10 } })
+    if (!fetchResult.ok) {
+      return true
+    }
+  const fetchJSON = await fetchResult.json() as any // eslint-disable-line
+    if (fetchJSON.files == null) {
+      return true
```

same, I think it should throw an error in this case